GuildedRoseLLCSpec/ItemCollectionViewDataSourceSpec.swift

```diff
@@ -14,10 +14,10 @@ class ItemCollectionViewDataSourceSpec: QuickSpec {
             return collectionView
         }
         
-        describe("the number of items") {
+        describe("building a datasource with a list of items") {
             it("has no items") {
                 let dataSource = ItemCollectionViewDataSource()
-                dataSource.show(items: [])
+                dataSource.show()
```

Comment:
Nice improvement on the test description. Non-blocking, but it might be better to say "has no items by default" since you changed the `show` to default `items = []` when none are provided.

Reply:
👍